ReactiveObjC.podspec

```diff
@@ -0,0 +1,62 @@
+Pod::Spec.new do |s|
+
+  s.name         = "ReactiveObjC"
+  s.version      = "1.0.0"
+  s.summary      = "The 2.x ReactiveCocoa Objective-C API: Streams of values over time"
+
+  s.description  = <<-DESC.strip_heredoc
+                     ReactiveObjC (formally ReactiveCocoa or RAC) is an Objective-C
+                     framework inspired by [Functional Reactive Programming](
+                     http://en.wikipedia.org/wiki/Functional_reactive_programming).
+                     It provides APIs for composing and **transforming streams of values**.
+                   DESC
+
+  s.homepage     = "https://reactivecocoa.io"
+  s.screenshots  = "https://reactivecocoa.io/img/logo.png"
+  s.license      = { type: "MIT", file: "LICENSE.md" }
+
+  s.documentation_url  = "https://github.com/ReactiveCocoa/ReactiveObjC/"\
+                         "tree/master/Documentation#readme"
+
+  s.author             = { "Josh Abernathy" => "josh@github.com" }
```

Comment:
Can you set the author to be `ReactiveCocoa` to match [what ReactiveSwift does](https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/ReactiveSwift.podspec).